system_metrics_collector/src/system_metrics_collector/main.cpp

```diff
@@ -59,7 +59,7 @@ int main(int argc, char ** argv)
     std::make_shared<system_metrics_collector::LinuxProcessMemoryMeasurementNode>(
     "linuxProcessMemoryCollector",
     1000ms,
-    "not_publishing_yet",
+    STATISTICS_TOPIC_NAME,
```

Comment:
This system metric viewer will receive process metrics as well and filter them out?

Reply:
That's my current plan, yes. The viewing problem can be split into the following:

1). Logging the data into a parseable format for a display tool
2). Parse the logged data
3). Plot the data